api_client/python/timesketch_api_client/error.py

```diff
@@ -27,3 +27,11 @@ def error_message(response, message=None, error=RuntimeError):
         text = soup.p.string
     raise error('{0:s}, with error [{1:d}] {2!s} {3:s}'.format(
         message, response.status_code, response.reason, text))
+
+
+class Error(Exception):
```

Comment:
Pretty generic name. Maybe ApiClientError?

Reply:
this is not an error to use... that iis this is following "proper way of doing things", see: https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions

```
When creating a module that can raise several distinct errors, a common practice is to create a base class for exceptions defined by that module, and subclass that to create specific exception classes for different error conditions:
```

This is how we have done things in other projects, eg. plaso, etc

Reply:
okm